web/style.css

```diff
@@ -45,20 +61,56 @@ footer {
 	justify-content: space-between;
 }
 
-footer a,
-footer span {
-	margin: 0 auto;
+.footer-links {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+}
+
+.footer-item {
+	margin: 1em 0;
+}
+
+p {
+	margin-bottom: 0;
+}
+
+.container {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
```

Comment:
in the old layout, once we're connected the code textbox is shoved to the bottom of the window, drawing attention away from it. i quite like that. it's no longer important so it should no longer occupy the middle of the screen. can we keep that effect? if not by layout then maybe by styling the disabled textbox with more subdued colours?

Reply:
Yeah definitely! I think that's a good feature, so I'll get it added back in :)